avro/swagger/src/main/java/com/cerner/beadledom/avro/SwaggerAvroModelConverter.java

```diff
@@ -119,49 +115,33 @@ protected String getName(Schema schema) {
    * determined merely by looking at the schema. It may also be used recursively to build collection
    * and union types.
    *
-   * @return the parsed property, or null if it cannot/should not be represented in the Swagger
-   *     model
+   * @return the parsed property, or null if it cannot/should not be represented in the Swagger model
    */
   @Nullable
-  protected ModelProperty parseSchema(Schema schema) {
+  protected Property parseSchema(Schema schema) {
     switch (schema.getType()) {
       case RECORD:
-        return simpleProperty(getName(schema), schema.getFullName());
+        return PropertyBuilder.build(getName(schema), schema.getFullName(), Collections.emptyMap());
```

Comment:
Don't all these types besides the UNION type need to be set as required in the swagger property?